pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -184,6 +192,18 @@ public BrokerResponse handleRequest(JsonNode request, @Nullable RequesterIdentit
       requestStatistics.setErrorCode(QueryException.PQL_PARSING_ERROR_CODE);
       return new BrokerResponseNative(QueryException.getException(QueryException.PQL_PARSING_ERROR, e));
     }
+    if (isLiteralOnlyQuery(brokerRequest)) {
+      LOGGER.info("Request {} contains only Literal, skipping server query: {}", requestId, query);
```

Comment:
This can potentially flood the log. Can we extract line 376-405 in existing code into a method and call that here?

Reply:
Changed this log to debug level.
For the code from `376-405`, I feel it makes too many assumptions which current broker query doesn't honor. I prefer to keep them separately.